MANIFEST.in

```diff
@@ -1,2 +1,4 @@
 include requirements.txt
-include CHANGELOG.rst
\ No newline at end of file
+include CHANGELOG.rst
+include lightning.qubit.toml
```

Comment:
Are we going to add `lightning.gpu` after our team discussion?

Reply:
Yes.

Reply:
Added `lightning_gpu.toml` here: https://github.com/PennyLaneAI/pennylane-lightning/pull/566/commits/ef2f1c523fb5b8ea11fdcaef0f2cde3d576d5452